prometheus.yml

```diff
@@ -0,0 +1,8 @@
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+scrape_configs:
+- job_name: curity
+  metrics_path: /metrics
+  static_configs:
+  - targets: ['idsvr:4466']
```

Comment:
does this host, `idsvr`, resolve? Should this be localhost or does something need to be done when starting Docker to allow Prometheus to find Curity? Maybe provide a Docker compose file instead of or in addition to the examples on how to start the various services using respective Docker commands

Reply:
In order for idsvr to resolve you will have to setup more. I added a Docker compose file and will update the documentation accordingly.